### PR TITLE
which => these

### PR DESCRIPTION
**Pull request by franzholz**
"which" would be a question or the second part of a sentence.

Releases: master, 10.4, 9.5

### CODE REVIEW

File: Documentation/ApiOverview/SiteHandling/Basics.rst [hunk 1]
```diff
@@ -9,17 +9,17 @@ Basics
 ======
 
 .. note::
-   Site handling as described here is available since TYPO3 9 LTS.
+   Site handling is available since TYPO3 9 LTS.
 
-TYPO3 site handling and configuration is the starting point for creating new web sites. The corresponding modules are found in the TYPO3 backend
-in the section "Site management".
+TYPO3 site handling and configuration is the starting point for creating new web sites. 
+The corresponding modules are found in the TYPO3 backend in the section "Site management".
```
Comment (mbrodala) [inline Documentation/ApiOverview/SiteHandling/Basics.rst:15]:
```suggestion
The corresponding modules are found in the TYPO3 backend in the section :guilabel:`Site management`.
```

File: Documentation/ApiOverview/SiteHandling/Basics.rst [hunk 2]
```diff
@@ -9,17 +9,17 @@ Basics
...
 
 A site configuration consists of the following parts:
 
-* Base URL configuration: Under which domain(s) is my site accessible
```
Comment (mbrodala) [inline Documentation/ApiOverview/SiteHandling/Basics.rst:19]: This actually was a question with question mark thus actually OK.

File: Documentation/ApiOverview/SiteHandling/Basics.rst [hunk 3]
```diff
@@ -9,17 +9,17 @@ Basics
...
-* :ref:`Language configuration<sitehandling-addingLanguages>`: Which languages are available for my site
-* :ref:`Error Handling<sitehandling-errorHandling>`: How should errors for this site behave (For example: configure custom 404 pages)
-* :ref:`Static Routes<sitehandling-staticRoutes>`: Add static routes to a site (For example for robots.txt on a per site base)
+* Base URL configurations: Under these domain(s) my site is accessible.
```
Comment (mbrodala) [inline Documentation/ApiOverview/SiteHandling/Basics.rst:19]:
```suggestion
* Base URL configurations: the domain(s) to access my site:
```

File: Documentation/ApiOverview/SiteHandling/Basics.rst [hunk 4]
```diff
@@ -9,17 +9,17 @@ Basics
...
+* :ref:`Language configuration<sitehandling-addingLanguages>`: These languages are available for my site.
```
Comment (mbrodala) [inline Documentation/ApiOverview/SiteHandling/Basics.rst:20]:
```suggestion
* :ref:`Language configuration<sitehandling-addingLanguages>`: the languages of my site.
```

File: Documentation/ApiOverview/SiteHandling/Basics.rst [hunk 5]
```diff
@@ -9,17 +9,17 @@ Basics
...
+* :ref:`Error Handling<sitehandling-errorHandling>`: How should errors for this site behave (For example: configuration of custom 404 pages)
```
Comment (mbrodala) [inline Documentation/ApiOverview/SiteHandling/Basics.rst:21]:
```suggestion
* :ref:`Error Handling<sitehandling-errorHandling>`: error behavior of my site (For example: configuration of custom 404 pages)
```

File: Documentation/ApiOverview/SiteHandling/Basics.rst [hunk 6]
```diff
@@ -9,17 +9,17 @@ Basics
...
+* :ref:`Static Routes<sitehandling-staticRoutes>`: Add static routes to a site (For example :file:`robots.txt` on a per site base)
```
Comment (mbrodala) [inline Documentation/ApiOverview/SiteHandling/Basics.rst:22]:
```suggestion
* :ref:`Static Routes<sitehandling-staticRoutes>`: static routes of my site (For example :file:`robots.txt` on a per site base)
```

**Review by mbrodala — CHANGES_REQUESTED** [review]
Found a few more things which could be improved while we're at it.

File: Documentation/ApiOverview/SiteHandling/Basics.rst (hunk 2 above)
Comment (franzholz) [reply Documentation/ApiOverview/SiteHandling/Basics.rst:19]: The answer stands before. So it cannot be a question any more.

File: Documentation/ApiOverview/SiteHandling/Basics.rst (hunk 5 above)
Comment (franzholz) [reply Documentation/ApiOverview/SiteHandling/Basics.rst:21]: Which English is to be used for TYPO3?

Comment (mbrodala) [reply Documentation/ApiOverview/SiteHandling/Basics.rst:21]: A good question but right now there is no strict rule here. Personally I'd try to aim at American English though.

**Review by mbrodala — APPROVED** [review]
Thanks a lot!

**Pull request merged**